src/library/masq.js

```diff
@@ -89,6 +105,24 @@ class Masq {
     this.masterKey = null
     this.peer = null
     this.app = null
+    this.dbName = null // used only during userApp registration
+
+    // init state
+    this.state = STATES.NOT_LOGGED
+  }
+
+  setState (newState) {
+    if (STATE_DEBUG) console.log(` ##### From ${this.state} -> ${newState} ######`)
+    switch (newState) {
```

Comment:
there is only one case, so we can just use an `if` here

Reply:
yes, the purpose was to use a case for multiple  states, but for now we only have one, so if is better.